google/cloud/storage/async/connection.h

```diff
@@ -85,7 +85,7 @@ class AsyncConnection {
     /// The name of the bucket and object to read. Includes optional parameters,
     /// such as pre-conditions on the read operation, or the range within the
     /// object to read.
```

Comment:
I presume the comment still holds despite the type change.

Reply:
Yes, it does.